package.json

```diff
@@ -78,7 +79,7 @@
     "@types/node": "^14.17.4",
     "@types/sinon": "^10.0.0",
     "@types/sinon-chai": "^3.2.5",
-    "axe-core": "^4.4.3",
+    "axe-core": "4.4.3",
```

Comment:
I had to pin this as it seems the axe version coming from playwright is not working with our karma setup.

Reply:
We probably should pin all our deps to be honest

Reply:
In bpmn-io land we always opted against this as it's not recommended for library providers (rationale: https://docs.renovatebot.com/dependency-pinning/#ranges-for-libraries, avoid duplicated libraries in the node modules). I'm not 100% sure what's best practice in the current ecosystem, but I'd still avoid pins until we really need it - as in this example.

For security reasons, I believe we should do housekeeping (`npm audit fix`) more often + utilize Renovate.